Neos.Flow/Classes/Http/Middleware/SessionMiddleware.php

```diff
@@ -37,30 +40,36 @@ class SessionRequestComponent implements ComponentInterface
      */
     protected $sessionManager;
 
-    /**
-     * @param ComponentContext $componentContext
-     */
-    public function handle(ComponentContext $componentContext)
+    public function process(ServerRequestInterface $request, RequestHandlerInterface $handler): ResponseInterface
```

Comment:
```suggestion
    public function process(ServerRequestInterface $request, RequestHandlerInterface $next): ResponseInterface
```
`$next` transports the information that this handler is the next in the chain and avoids the awkward `$handler->handle()` invocation